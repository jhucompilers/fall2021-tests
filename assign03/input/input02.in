PROGRAM input02;
    TYPE Big = ARRAY 1000 OF INTEGER;
    VAR big: Big;
BEGIN
    READ big[0];
    READ big[big[0]];
    READ big[big[big[0]]];
END.