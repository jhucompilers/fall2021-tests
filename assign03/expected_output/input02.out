0,TYPE,Big,ARRAY 1000 OF INTEGER
0,VAR,big,ARRAY 1000 OF INTEGER
